datahub-actions/setup.py

```diff
@@ -33,7 +33,7 @@ def get_long_description():
 acryl_datahub_min_version = os.environ.get("ACRYL_DATAHUB_MIN_VERSION") or "0.12.1.2"
 
 base_requirements = {
-    f"acryl-datahub[kafka]>={acryl_datahub_min_version}",
+    f"acryl-datahub[datahub-kafka]>={acryl_datahub_min_version}",
```

Comment:
It seems kafka and datahub-kafka both used `kafka_common` from `metadata-ingestion/setup.py`, how would this change support pydantic 2? I looked at the gradle build log for this PR and it looks like it first pulls pydantic `pydantic-2.5.3` and then uninstalls it and installs `pydantic-1.10.14` instead.

Reply:
All plugins except for a couple currently get a pydantic v1 pin https://github.com/datahub-project/datahub/blob/1498c36875450b1a1f44d53e8e8c47c41a91dc69/metadata-ingestion/setup.py#L737

I believe it installs pydantic v1 in the gradle build because it includes the snowflake_tag_propagation action that needs the acryl-datahub snowflake plugin, which needs pydantic v1

However, this should allow other usages of actions to not need pydantic v1